CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 20221213
+- ecs version: 1.67.2-1
+- source al2022 ami: al2022-ami-minimal-2022.0.20221207.4-kernel-5.15-x86_64
+- source al2022 arm ami: al2022-ami-minimal-2022.0.20221207.4-kernel-5.15-arm64
+- distribution al2022 release: 2022.0.20221207
```

Comment:
maybe we don't need to add `distribution al2022 release` in the changelog for `20221213` release as it's the same in the previous  `20221207` release.